packages/website/src/components/editor/createProvideCodeActions.ts

```diff
@@ -34,9 +34,9 @@ export function createProvideCodeActions(
               edits: [
                 {
                   resource: model.uri,
-                  // @ts-expect-error monaco for ts >= 4.8
+                  // monaco for ts >= 4.8
                   textEdit: editOperation,
-                  // monaco for ts < 4.8
+                  // @ts-expect-error monaco for ts < 4.8
```

Comment:
i pushed lint fix to v6 for this and other issues that where caused by incorrect merge, should be gone after rebase/merge